lib/galaxy/model/mapping.py

```diff
@@ -985,6 +985,7 @@
     Column("from_path", TEXT),
     Column("published", Boolean, index=True, default=False),
     Index('ix_stored_workflow_slug', 'slug', mysql_length=200),
+    Index('ix_stored_workflow_from_path', 'from_path', mysql_length=200),
```

Comment:
That doesn't look like a super-useful index, I wouldn't add this.

Reply:
Removing.